web/src/components/OrderInput/index.tsx

```diff
@@ -71,13 +87,16 @@ export const OrderInput: React.FunctionComponent<OrderInputProps> = React.memo((
                     inputValue={value}
                     placeholder={placeholder || '0'}
                     handleChangeInput={handleChangeValue}
-                    label={value && label ? label : ''}
-                    defaultLabel={value && label ? label : ''}
+                    label={(value || labelVisible) && label ? label : ''}
+                    labelVisible={labelVisible}
+                    defaultLabel={(value || labelVisible) && label ? label : ''}
                     onKeyPress={onKeyPress}
                     handleFocusInput={() => handleFocusInput(props.label)}
+                    autoFocus={autoFocus}
                 />
             </fieldset>
             <div className={cryptoIconClass}>
+                { showAllButton && <span className="cr-order-input__crypto-icon-all" onClick={handleClickAllButton}>ALL</span> }
```

Comment:
ALL should be added to translation files

Reply:
Fixed, thanks.